js/index.js

```diff
@@ -40,3 +40,60 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// Task 1: Create selectors to point your data into elements
+
+const navBar = document.querySelector('nav');
+const anchorTags = document.querySelectorAll('a');
+const mainBtn = document.querySelector('button');
+const mainHeader = document.querySelector('h1');
+const codeImage = document.getElementById('cta-img');
+const h4Tags = document.querySelectorAll('h4');
+const pTags = document.querySelectorAll('p');
+const middleImg = document.getElementById('middle-img');
+
+
+//Task 2: Update HTML with JSON data
+
+anchorTags[0].textContent = siteContent["nav"]["nav-item-1"];
+anchorTags[1].textContent = siteContent["nav"]["nav-item-2"];
+anchorTags[2].textContent = siteContent["nav"]["nav-item-3"];
+anchorTags[3].textContent = siteContent["nav"]["nav-item-4"];
+anchorTags[4].textContent = siteContent["nav"]["nav-item-5"];
+anchorTags[5].textContent = siteContent["nav"]["nav-item-6"];
+
+anchorTags.forEach(x => x.style.color = "teal");
+
+const newFirstAnchorTag = document.createElement('a');
+const newLastAnchorTag = document.createElement('a');
+newFirstAnchorTag.textContent = "Welcome";
+newLastAnchorTag.textContent = "Blog";
+navBar.prepend(newFirstAnchorTag);
+navBar.append(newLastAnchorTag);
+newFirstAnchorTag.style.color = "teal";
+newLastAnchorTag.style.color = "teal";
+
+mainHeader.textContent = siteContent["cta"]["h1"];
+mainBtn.textContent = siteContent["cta"]["button"];
+codeImage.src = "img/header-img.png";
+
+h4Tags[0].textContent = siteContent['main-content']['features-h4'];
+h4Tags[1].textContent = siteContent['main-content']['about-h4'];
+h4Tags[2].textContent = siteContent['main-content']['services-h4'];
+h4Tags[3].textContent = siteContent['main-content']['product-h4'];
+h4Tags[4].textContent = siteContent['main-content']['vision-h4'];
+
+pTags[0].textContent = siteContent['main-content']["features-content"];
+pTags[1].textContent = siteContent['main-content']["about-content"];
+pTags[2].textContent = siteContent['main-content']["services-content"];
+pTags[3].textContent = siteContent['main-content']["product-content"];
+pTags[4].textContent = siteContent['main-content']["vision-content"];
+
+middleImg.src = "img/mid-page-accent.jpg";
```

Comment:
You needed to refer to the object here: `siteContent["main-content"]["middle-img"]`